style.css

```diff
@@ -0,0 +1,178 @@
+body {
+    font-family: 'Roboto', sans-serif;
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 20px;
+    border-bottom: 1px solid lightgrey;
+    background-color: #ffffff;
+    box-shadow: 0 4px 10px -2px lightgray;
+}
+
+body {
+    background-color: #f8fafc;
+}
+
+.logos {
+    width: 20px;
+    height: 20px;
+    margin: 0.5em;
+}
+
+span {
+    color: #645cff;
+}
```

Comment:
Not sure if this is a good idea. The use case is very specific, but this will style all spans in the app (imagine if we'd scale it)